app/models/application_qualification.rb

```diff
@@ -94,6 +101,12 @@ def incomplete_degree_information?
     false
   end
 
+  def enic_reference=(value)
```

Comment:
How do you we feel about adding validation for the presence of the `enic_reason`? This would make sure that all new and edited records have this value set and will make it easier to set the column to `null: false` in the future?